src/QsCompiler/Transformations/IntrinsicMappingTransformation.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Quantum.QsCompiler.SyntaxTree;
+using System.Collections.Immutable;
+using System.Linq;
+
+
+namespace Microsoft.Quantum.QsCompiler.Transformations.IntrinsicMappingTransformation
+{
+    public class IntrinsicMappingTransformation
```

Comment:
How about renaming it to something like InstructionSetReplacement (mapping to me sounds like it's tied to a geometry rather than an instruction set)?

Reply:
See other conversation regarding naming.